homeassistant/components/refoss/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+  "domain": "refoss",
+  "name": "Refoss",
+  "codeowners": ["@ashionky"],
+  "config_flow": true,
+  "iot_class": "local_push",
+  "requirements": ["refoss-ha==1.0.3"]
```

Comment:
I could not find the public repository providing this package. Could you point us to it?

Reply:
May I ask if it is the Github address or Pypi?

Reply:
Repository... GitHub.

Reply:
thanks！  https://github.com/ashionky/refoss_ha

Reply:
resolved